### PR TITLE
Get rid of mocha tests in actionpack - part 2

### PR DESCRIPTION
**Pull request by rodzyn**
Here is the part 2, I think part 3 will be the last one that is completely removing all the leftovers of mocha

/cc @kaspth

### CODE REVIEW

File: actionpack/test/controller/integration_test.rb [hunk 1]
```diff
@@ -340,14 +386,8 @@ def method_missing(name, *args)
 # Tests that integration tests don't call Controller test methods for processing.
 # Integration tests have their own setup and teardown.
 class IntegrationTestUsesCorrectClass < ActionDispatch::IntegrationTest
-  def self.fixture_table_names
```
Comment (rodzyn) [inline actionpack/test/controller/integration_test.rb]: This is not needed anymore for tests to pass

File: actionpack/test/controller/integration_test.rb [hunk 2]
```diff
@@ -340,14 +386,8 @@ def method_missing(name, *args)
...
-    []
-  end
-
   def test_integration_methods_called
     reset!
-    @integration_session.stubs(:generic_url_rewriter)
```
Comment (rodzyn) [inline actionpack/test/controller/integration_test.rb]: This is not needed anymore for tests to pass

File: actionpack/test/dispatch/debug_exceptions_test.rb [hunk 3]
```diff
@@ -71,14 +71,6 @@ def call(env)
     end
   end
 
-  def setup
-    app = ActiveSupport::OrderedOptions.new
```
Comment (rodzyn) [inline actionpack/test/dispatch/debug_exceptions_test.rb]: This is not needed anymore for tests to pass

File: actionpack/test/dispatch/routing/inspector_test.rb [hunk 4]
```diff
@@ -12,12 +12,6 @@ module Routing
     class RoutesInspectorTest < ActiveSupport::TestCase
       def setup
         @set = ActionDispatch::Routing::RouteSet.new
-        app = ActiveSupport::OrderedOptions.new
```
Comment (rodzyn) [inline actionpack/test/dispatch/routing/inspector_test.rb]: This is not needed anymore for tests to pass

File: actionpack/test/dispatch/routing/inspector_test.rb [hunk 5]
```diff
@@ -316,9 +310,6 @@ def test_regression_route_with_controller_regexp
 
       def test_inspect_routes_shows_resources_route_when_assets_disabled
         @set = ActionDispatch::Routing::RouteSet.new
-        app = ActiveSupport::OrderedOptions.new
```
Comment (rodzyn) [inline actionpack/test/dispatch/routing/inspector_test.rb]: This is not needed anymore for tests to pass

**Comment by kaspth** [comment]
Thanks

**Pull request merged**